x-pack/plugins/maps/public/classes/sources/ems_file_source/ems_file_source.tsx

```diff
@@ -123,11 +123,7 @@ export class EMSFileSource extends AbstractVectorSource implements IEmsFileSourc
 
   async getGeoJsonWithMeta(): Promise<GeoJsonWithMeta> {
     const emsFileLayer = await this.getEMSFileLayer();
-    const featureCollection = await fetchGeoJson(
-      emsFileLayer.getDefaultFormatUrl(),
-      emsFileLayer.getDefaultFormatType() as FORMAT_TYPE,
-      'data'
-    );
+    const featureCollection = (await emsFileLayer.getGeoJson()) ?? EMPTY_FEATURE_COLLECTION;
```

Comment:
This changes the behavior slightly.

I'd throw when `getGeoJson` returns nulll, instead of returning the `EMPTY_FEATURE_COLLECTION`, to preserve the original flow. Otherwise, errors don't propagate to the layer-UX (the little warning triangle in the TOC) as they would earlier (e.g. because network connection issue, faulty map-config, ...)